packages/nemo-evaluator/src/nemo_evaluator/api/api_dataclasses.py

```diff
@@ -114,6 +114,23 @@ class EvaluationConfig(BaseModel):
     type: Optional[str] = Field(description="Type of the task", default=None)
 
 
+class EvaluationMetadata(BaseModel):
```

Comment:
Do we need a pre-defined structure here? If we are to pass metadata maybe it would make sense to allow end users to pass their own as well. That is, I would like to keep SDK and launcher de-coupled. Wonder what you think about that.

In such case, metadata would be just of type `dict`. Please note as both these fields are optional, we are not validating anything but we do block other metadata to be easily injected